upgrade_guide/README.md

```diff
@@ -1,5 +1,60 @@
 ## Upgrade guide
 
+### 4.17.0
+
+#### Java17 is now a supported platform
+
+With [JAVA-3042](https://datastax-oss.atlassian.net/browse/JAVA-3042) the driver is now tested against Java17 as
+well as the other LTS JVM releases Java8 and Java11.  Based on our testing the Java driver appears to function
+normally on Java17, but of course bugs are always a possibility.  Issues arising from the Java17 platform will be
+triaged and addressed just like any other issue.
```

Comment:
I'm not super-happy with the way this language came out.  If anybody has any suggestions on a way to say all this in something that sounds more like English I'm all ears.

Reply:
I'm happy with this from a technical sentiment pov